docs/content/Configuration/VPC/Connecting-with-a-VPC-AWS.mdx

```diff
@@ -5,8 +5,8 @@ permalink: /cloud/configuration/connecting-with-a-vpc/aws
 
```

Comment:
It would be nice to prepend this and similar pages (and, honestly, almost every page in the docs) with a paragraph that outlines what is described on the page. For example:

```suggestion
To connect with a VPC on AWS, you need to collect the necessary information and hand it over to the Cube Cloud representative. Next, you'll have to accept the VPC peering request and configure security groups and route tables.

```